README.md

```diff
@@ -207,6 +208,26 @@ The following documentation is available:
 * [Release Notes](https://github.com/minishift/minishift/releases)
 * [Developing Minishift](./docs/developing.md)
 
+<a name="using-minishift-with-a-proxy"></a>
```

Comment:
I think this really should be in _using.md_ and not in the README. First we don't want to get the README to long and secondly we want this to be part of the generated docs later to be hosted on docs.openshift.org. As part of that we most likely won't include the README. That is specific to the GitHub repo.